python/lsst/cp/pipe/makeBrighterFatterKernel.py

```diff
@@ -263,6 +264,10 @@ def run(self, inputPtc, dummy, camera, inputDims):
                 if q[0][0] > 0.0:
                     self.log.warning("Amp: %s %d skipped due to value of (variance-mean)=%f",
                                      ampName, xcorrNum, q[0][0])
+                    # If we drop an element of ``scaledCorrList``
+                    # (which is what this does), we need to ensure we
+                    # drop the flux entry as well.
+                    fluxes = np.delete(fluxes, xcorrNum)
```

Comment:
No, this delete is needed so that the length of the cross-correlations and fluxes match when we pass them to the quadratic solver.  The `mask` and `varaiances` aren't passed anywhere, so they can be left alone.